pkg/proctree/proctree_procfs.go

```diff
@@ -149,7 +149,7 @@ func dealWithProc(pt *ProcessTree, givenPid int32) error {
 
 	// sanity checks
 	switch givenPid {
-	case 0, 1, 2: // PIDs 0, 1 and 2 are special
+	case 1, 2: // PIDs 1 and 2 are special
```

Comment:
Something here is off. givenPid is checked at line 119, which already short circuits 0 pid. But it's again here and in [169](https://github.com/aquasecurity/tracee/pull/4830/files#diff-d7f4ec36e320ee8375b526e3d34325333860ade46fface1f448bcb69f4e683bcR169). I wonder about the original rationale... those sanity checks were really intended to the givenPid or to discovered based on the given?

Reply:
I agree this is not clear. I also verified that dealWithProc function can never get givenPid with 0 value, so even the check at line 119 does nothing...